src/darker/__main__.py

```diff
@@ -213,9 +213,10 @@ def _blacken_and_flynt_single_file(
         len(fstringified.lines),
         "some" if has_fstring_changes else "no",
     )
-    # 3. run black on the isorted and fstringified contents of each edited to-file
-    formatted = _maybe_blacken_single_file(
-        relative_path_in_rev2, exclude.black, fstringified, black_config
+    # 3. run the code re-formatter on the isorted and fstringified contents of each
+    #    edited to-file
```

Comment:
Could we add a debug log showing which formatter we are using

Reply:
We now have
```python
    logger.debug(
        "Running %r by %s.%s resulted in %s changed lines within a total of %s lines",
        formatter.name,
        formatter.__module__,
        type(formatter).__name__,
        "no" if formatted == fstringified else "some",
        len(formatted.lines),
    )
```